README.md

```diff
@@ -1,36 +1,41 @@
-# Yo Azure Data Studio - Extension and Customization Generator
+# Azure Data Studio - Extension and Customization Generator
```

Comment:
For consistency sake across documentation, do we want to rename this just the 'Azure Data Studio Extension Generator'?